### DISCUSSION TITLE
[5]PETSC ERROR: Caught signal number 15 Terminate: Some process (or the batch system) has told this process to end

### DISCUSSION THREAD

**Discussion by usmanshahid90** [mdolab/dafoam#391] [Bug]
Hi,
I am running Aircraft optimization case with the given tutorial . I am having signal number 15 error during first sensitivity run. Kindly guide me. Log file is attached below.

Thanks

[Signal 15 Error.txt](https://github.com/mdolab/dafoam/files/10486742/Signal.15.Error.txt)

**Comment by friedenhe** [comment]
Check FAQ and see if you have enough memory to run this case.
